modules/network-observability-configuring-FLP-sampling.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+
+// * configuring-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-config-FLP-sampling{context}"]
+
+= Configuring the Flow Collector resreouce
+As an alternative to editing YAML in the {product-title} web console, you can do configure specifications, such as eBPF sampling, by patching the `flowcollector` custom resource definition (CRD):
+
+.Procedure
+
+. Run the following command to patch the `flowcollector` CRD and update the `spec.agent.ebpf.sampling` value:
```

Comment:
Maybe it's nitpicking but the users can edit the FlowCollector CR (custom resource) which is the configuration, and not the CRD, which is the schema. I haven't checked other parts of the doc if the terms are used strictly or interchangeably

Reply:
(comment also applies above: '[...] by patching the `flowcollector` custom resource (CR)')